sellercenter-product-port/pom.xml

```diff
@@ -70,6 +70,16 @@
           <version>3.3.3</version>
           <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.mock-server</groupId>
+        <artifactId>mockserver-netty</artifactId>
```

Comment:
do you think that we really need to use mock server instead of mocking JsonReader?

Reply:
There is no way to do it. You can take a look in history, there was a test for getting list of products. But then instead of reading whole response to one line we started to use some performance optimizations. JsonReader is not even in use in my new method and other one which gets list of products. 
There is one way of doing test through mocking dependencies - by injecting productMapper. But then significant part of what I _really_ wanted to test will be lost. In particular I wanted to test that with given search parameters code will execute HTTP request with expected parameters.